public/theme/skeleton/custom/setcontent_1.twig

```diff
@@ -155,7 +155,7 @@
         {% endfor %}
 
         <ul>
-            <li>Two different random results: [{{ first_ids === second_ids ? "no" : "yes" }}]</li>
+            <li>Two different random results: [{{ first_ids === second_ids ? "[no]" : "[yes]" }}]</li>
```

Comment:
Are the double brackets around `[[yes]]` or `[[no]]` intentional?

Reply:
nope! definitely not.